web/hooks/push_test.go

```diff
@@ -0,0 +1,31 @@
+package hooks
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestExtractChanges(t *testing.T) {
```

Comment:
Doesn't look like it is related to push functionality. Please rename to `extract_changes_test.go` and correspondingly for the relevant functions (from other files) should be renamed to `extract_changes.go`